src/govuk/components/modal-dialogue/_modal-dialogue.scss

```diff
@@ -0,0 +1,161 @@
+@import "../../settings/all";
+@import "../../tools/all";
+@import "../../helpers/all";
+
+@include govuk-exports("govuk/component/modal-dialogue") {
+  $govuk-dialogue-width: 640px;
+
+  .govuk-modal-dialogue,
+  .govuk-modal-dialogue__backdrop {
+    position: fixed;
+    z-index: 0;
```

Comment:
Increase to 1000 so that this will site above all other elements already in the site.

I had to do this when I've used this code as several elements floated on top of the backdrop.